app/jobs/search_keyword_job.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class SearchKeywordJob < ApplicationJob
+  attr_reader :keyword
+
+  def perform(keyword_id)
+    @keyword = Keyword.includes(:source).find(keyword_id)
+    search_result = search_service
+    update_keyword(search_result)
+  end
+
+  private
+
+  def search_service
+    source_name = keyword.source.name.downcase
+    keyword_name = keyword.name
```

Comment:
it's good idea to remove the duplication of the `keyword.name`. However, either way, it just call one time because on the of condition of the source_name.